src/sidebar/components/test/selection-tabs-test.js

```diff
@@ -1,245 +1,249 @@
+      fakeSettings.enableExperimentalNewNoteButton = true;
+      const wrapper = createComponent({
+        selectedTab: uiConstants.TAB_NOTES,
+        totalNotes: 0,
+      });
       assert.include(
-        unavailableTutorial.textContent,
+        wrapper.find('.annotation-unavailable-message__tutorial').text(),
         'Create one by clicking the'
       );
-      assert.isTrue(noteIcon.classList.contains('h-icon-note'));
+      assert.isTrue(
+        wrapper
+          .find('.annotation-unavailable-message__tutorial i')
+          .hasClass('h-icon-note')
+      );
     });
 
     it('should display the longer version of the no annotations message when there are no annotations', function() {
-      fakeSession.state.preferences.show_sidebar_tutorial = false;
-      fakeSettings.enableExperimentalNewNoteButton = false;
-
-      const elem = util.createDirective(document, 'selectionTabs', {
-        selectedTab: 'annotation',
+      const wrapper = createComponent({
         totalAnnotations: 0,
-        totalNotes: '10',
       });
-      const unavailableMsg = elem[0].querySelector(
-        '.annotation-unavailable-message__label'
-      );
-      const unavailableTutorial = elem[0].querySelector(
-        '.annotation-unavailable-message__tutorial'
-      );
-      const noteIcon = unavailableTutorial.querySelector('i');
-
       assert.include(
-        unavailableMsg.textContent,
+        wrapper.find('.annotation-unavailable-message__label').text(),
         'There are no annotations in this group.'
       );
       assert.include(
-        unavailableTutorial.textContent,
+        wrapper.find('.annotation-unavailable-message__tutorial').text(),
         'Create one by selecting some text and clicking the'
       );
-      assert.isTrue(noteIcon.classList.contains('h-icon-annotate'));
+      assert.isTrue(
+        wrapper
+          .find('.annotation-unavailable-message__tutorial i')
+          .hasClass('h-icon-annotate')
+      );
     });
 
     context('when the sidebar tutorial is displayed', function() {
-      fakeSession.state.preferences.show_sidebar_tutorial = true;
-
       it('should display the shorter version of the no notes message when there are no notes', function() {
-        const elem = util.createDirective(document, 'selectionTabs', {
-          selectedTab: 'note',
-          totalAnnotations: '10',
+        fakeSession.state.preferences.show_sidebar_tutorial = true;
+        const wrapper = createComponent({
           totalNotes: 0,
+          selectedTab: uiConstants.TAB_NOTES,
         });
-        const msg = elem[0].querySelector(
-          '.annotation-unavailable-message__label'
-        );
 
-        assert.include(
-          msg.textContent,
-          'There are no page notes in this group.'
-        );
-        assert.notInclude(msg.textContent, 'Create one by clicking the');
+        const msg = wrapper.find('.annotation-unavailable-message__label');
```

Comment:
If a selector is repeated frequently in tests, I find it can be helpful to extract that into a helper function (eg. `const unavailableMessage = wrapper => wrapper.find(...)`), so there is less work to do to update the tests if you decide to later change the HTML structure.